src/test/resources/mgmt/client.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "My 1 application",
+  "description": "My 1 application",
```

Comment:
There's no reason to change this JSON files content unless you're going to test them. If so, please use a different string that the one used for the `name`. This applies to the other file as well.